lexos/models/stats_model.py

```diff
@@ -0,0 +1,179 @@
+        """Converts word lists completely to statistic."""
+        assert np.sum(self._doc_term_matrix.values) > 0, EMPTY_LIST_MESSAGE
+        # initialize
+        file_anomaly_iqr = {}
+        file_anomaly_std_err = {}
+        num_file = np.size(self._doc_term_matrix.index.values)
+        file_sizes = np.sum(self._doc_term_matrix.values, axis=1)
+        labels = [self._id_temp_label_map[file_id]
+                  for file_id in self._doc_term_matrix.index.values]
+
+        # 1 standard error analysis
+        average_file_size = round(np.average(file_sizes), 3)
+        # Calculate the standard deviation
+        std_dev_file_size = np.std(file_sizes).item()
+        # Calculate the anomaly
+        for count, label in enumerate(labels):
+            if file_sizes[count] > average_file_size + 2 * std_dev_file_size:
+                file_anomaly_std_err.update({label: 'large'})
+            elif file_sizes[count] < average_file_size - 2 * std_dev_file_size:
+                file_anomaly_std_err.update({label: 'small'})
+
+        # 2 iqr analysis
+        median = np.median(file_sizes).item()
+        q1 = np.percentile(file_sizes, 25, interpolation="midpoint")
+        q3 = np.percentile(file_sizes, 75, interpolation="midpoint")
+        iqr = q3 - q1
+
+        # calculate the anomaly
+        for count, label in enumerate(labels):
+            if file_sizes[count] > median + 1.5 * iqr:
+                file_anomaly_iqr.update({label: 'large'})
+            elif file_sizes[count] < median - 1.5 * iqr:
+                file_anomaly_iqr.update({label: 'small'})
+
+        return CorpusInfo(q1=q1,
+                          q3=q3,
+                          iqr=iqr,
+                          median=median,
+                          average=average_file_size,
+                          num_file=num_file,
+                          file_sizes=list(file_sizes),
+                          file_names=labels,
+                          anomaly_iqr=file_anomaly_iqr,
+                          std_deviation=std_dev_file_size,
+                          anomaly_std_err=file_anomaly_std_err)
+
+    @staticmethod
+    def _get_file_info(count_list: np.ndarray, file_name: str) -> FileInfo:
+        """Gives statistics of a particular file in a given file list.
+
+        :param count_list: a list contains words count of a particular file.
+        :param file_name: the file name of that file.
+        """
+        assert np.sum(count_list) > 0, EMPTY_LIST_MESSAGE
+        # initialize remove all zeros from count_list
+        nonzero_count_list = count_list[count_list != 0]
+        num_word = np.size(nonzero_count_list)
+        total_word_count = int(sum(nonzero_count_list).item())
+        # 1 standard error analysis
+        average_word_count = round(total_word_count / num_word, 3)
+        # calculate the standard deviation
+        std_word_count = np.std(nonzero_count_list).item()
+
+        # 2 iqr analysis
```

Comment:
And I'm assuming "1 standard error" and "2 iqr analysis" is you labeling steps, but that's a little unclear (maybe "2 iqr analysis" is a specific technique). You could write "# 2) stuff" or "# 2. stuff"